rle_array/autoconversion.py

```diff
@@ -86,3 +101,30 @@ def auto_convert_to_rle(df: pd.DataFrame, threshold: float = 1.0) -> pd.DataFram
         data[col] = array_target
 
     return pd.DataFrame(data, index=index)
+
+
+def decompress(df: pd.DataFrame, threshold: Optional[float] = None) -> pd.DataFrame:
```

Comment:
threshold is an unused argument(and doesn't really make sense for decompression)